release-groups.txt

```diff
@@ -3006,4 +3010,4 @@ Zuzuu
 ZX
 ZZGtv
 ~AA~
-ßsZ
\ No newline at end of file
+ßsZ
```

Comment:
What's going on here? Please make sure the the commit doesn't contain any unnecessary changes.